.github/settings.yml

```diff
@@ -1,6 +1,12 @@
 repository:
   default_branch: master
 
+branches:
+  - name: release*
```

Comment:
@claudiahdz and I had a convo about this and we think it should be changed to `*`